src/scripts/main.js

```diff
@@ -1 +1,76 @@
 'use strict';
+
+function showMessege(message, className) {
+  const el = document.createElement('div');
+
+  el.setAttribute('data-qa', 'notification');
+  el.classList.add(className);
+  el.textContent = message;
+
+  document.body.appendChild(el);
+}
+
+let rightClick = false;
+let leftClick = false;
+
+const firstPromise = new Promise((resolve, reject) => {
+  document.addEventListener('click', (e) => {
+    if (e.button === 0) {
+      resolve('First promise was resolved');
+    }
+  });
+
+  setTimeout(() => {
+    reject(new Error('First promise was rejected'));
+  }, 3000);
+});
+
+const secondPromise = new Promise((resolve) => {
+  document.addEventListener('click', (e) => {
+    if (e.button === 0) {
+      rightClick = true;
+    }
+
+    if (rightClick || leftClick) {
+      resolve('Second promise was resolved');
+    }
```

Comment:
The logic here might not work as intended. The `rightClick` variable is set to `true` only when a right-click event occurs, but the promise resolves on any click if either `rightClick` or `leftClick` is true. This could lead to unexpected behavior. Consider revising the logic to ensure it aligns with the intended functionality.